ga4gh/backend.py

```diff
@@ -192,6 +198,68 @@ def _getEnd(cls, variant):
         return variant.end
 
 
+class VariantAnnotationsIntervalIterator(IntervalIterator):
+    """
+    An interval iterator for annotations
+    """
+
+    def __init__(self, request, parentContainer):
+        super(VariantAnnotationsIntervalIterator, self).__init__(
+            request, parentContainer)
+        print('self._request.featureIds: ', self._request.featureIds)
+        if self._request.featureIds is None:
+            self._featureIds = []
+        else:
+            self._featureIds = set(self._request.featureIds)
+        self._effects = set(self._request.effects)
```

Comment:
Seems to work OK for me:

   ./client_dev.py --verbose variantannotations-search --referenceName 1 --start 60000 --end 70000 http://localhost:8000/ | wc
    7064   12545  213964

No errors at all. How are you testing this? 
Can you send the command you are using?